CHANGELOG.md

```diff
@@ -81,6 +81,12 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a Ch
 
 ## v2.5.0 (2022-08-24)
 
+### BREAKING CHANGES
```

Comment:
We should note that this was an unintentional breakage, that we don't intend for minor version bumps to cause any breaking changes, and that this section was added retrospectively.